eng/helix/helix.proj

```diff
@@ -21,6 +21,7 @@
     <EnableAzurePipelinesReporter>false</EnableAzurePipelinesReporter>
     <IsExternal>true</IsExternal>
     <Creator>aspnetcore</Creator>
+    <SkipInvalidConfigurations>true</SkipInvalidConfigurations>
```

Comment:
We should circle back with the helix sdk folks if we have to turn this back on, its this issue, we can reopen it to track if need https://github.com/dotnet/arcade/issues/1605